_includes/cohort-data.html

```diff
@@ -0,0 +1,34 @@
+{% assign cohort = include.cohort %}
+{% assign people = site.data.people %}
+{% assign projects = site.data.cohorts.[cohort].projects %}
```

Comment:
Is the `.` between `cohorts` and `[cohort]` required? In other files, you did not put it. It would be good to be consistent there

Reply:
😮 Normally, the dot would be necessary to access keys...but is not needed since the cohort is already stated in the square brackets => [] 
Thank you for pointing this out. 🙏